src/assets/styles/global/_mixins.scss

```diff
@@ -10,3 +10,11 @@
     align-items: $align;
   }
 }
+@mixin mx-mobile-only($value){
+  margin-right: $value;
+  margin-left: $value;
+  @include md {
```

Comment:
now as I think about it it would be nice to make these mixins a bit more descriptive. I'd suggest to change the name schema to 'breakpoint-md' 'breakpoint-lg' etc.

Reply:
I'm not sure if in this case I can name the mixin by a breakpoint defined in the project. In this case the margin will be assigned when screen width Is less than 700px. There is no breakpoint with max-width: 700px